tests/integration/exec/php/src/last_payload.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if(isset($_GET['abc'])){
+
+    if(strpos($_GET['abc'], '&') !== false && strpos($_GET['abc'], ';') === false){
+        $output = system("echo foo &" . $_GET['abc']."& echo done");
```

Comment:
```suggestion
        $output = system("echo foo &" . $_GET['abc'] . "& echo done");
```

Je ne comprend pas l'intention

Reply:
le but est d'aller chercher le dernier payload de Wapiti dans ce module, ce payload est le suivant: ``&ping -n 25 127.0.0.1&``
La seule solution que j'ai trouvé pour aller le chercher est de sanitizer partiellement l'entrée comme le ferais un serveur mal configuré, et ce payload peut s'injecter dans un && en Bash